clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -9644,6 +9644,9 @@ void SYCLPostLink::ConstructJob(Compilation &C, const JobAction &JA,
   // Process device-globals.
   addArgs(CmdArgs, TCArgs, {"-device-globals"});
 
+  // Process host pipes.
+  addArgs(CmdArgs, TCArgs, {"-host-pipes"});
```

Comment:
We are planning on moving the CompileTimePropertiesPass to be an LLVM pass rather than a sub-feature of sycl-post-link, in which case we won't need this. See https://github.com/intel/llvm/pull/7527. Note also this may affect the sycl-post-link related changes in this PR, but it should be resolvable.

Reply:
@steffenlarsen Now that the CompileTimePropertiesPass change was merged, I rebased, moved my host pipe change out of sycl-post-link, and removed the host-pipe arg from the driver.